gunicorn/glogging.py

```diff
@@ -199,7 +199,9 @@ def setup(self, cfg):
                 self.access_log, cfg, self.syslog_fmt, "access"
             )
 
-        if cfg.logconfig:
+        if cfg.logconfigdict:
+            dictConfig(cfg.logconfigdict)
```

Comment:
I misunderstood your comment earlier, but I've implemented this now (i.e. the defaults will be applied regardless of whether dictConfig or fileConfig is used). As far as I can tell the format is the same for both.